base/8.15-jessie/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM node:8.15-jessie
+
+RUN apt-get update --fix-missing && \
+  apt-get install -y --no-install-recommends \
+    libgtk2.0-0 \
+    libnotify-dev \
+    libgconf-2-4 \
+    libnss3 \
+    libxss1 \
+    libasound2 \
+    xvfb \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
this line saves us 150mb

![image](https://user-images.githubusercontent.com/4010591/55578900-60a25500-56ed-11e9-91f5-33255da117c6.png)